### PR TITLE
Refactor: index.js format improvements

### PR DESCRIPTION
**Pull request by lucasrmendonca**
- Apply Prettier code formatter defaults.
- Improve some variable names for better clarity;
- Add missing "await"s in async functions;
- Use object destructuring when available;

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,61 +44,68 @@ async function launchWebAuthFlow({
...
   alwaysUseTab = false,
-  windowOptions
+  windowOptions,
 }) {
-  const wInfo = await createWindow({
-    type: "popup",
-    url,
-    state: "minimized",
+  const authWindow = await createWindow(
+    { type: "popup", url, state: "minimized", ...windowOptions },
```
Comment (eight04) [inline index.js:50]: This is a bad change. Comment below loses its context.

File: index.js [hunk 2]
```diff
@@ -44,61 +44,68 @@ async function launchWebAuthFlow({
...
+    ["blocking"]
+  );
   browser.webNavigation.onDOMContentLoaded.addListener(onDOMContentLoaded);
   browser.tabs.onRemoved.addListener(onTabRemoved);
+
   try {
     return await promise;
   } finally {
-    cleanup();
+    await cleanup();
```
Comment (eight04) [inline index.js:71]: This is wrong. Cleanup shouldn't block the main process.

File: index.js [hunk 3]
```diff
@@ -44,61 +44,68 @@ async function launchWebAuthFlow({
...
-  
-  function onDOMContentLoaded(details) {
-    if (details.frameId || details.tabId !== tabId) return;
+
+  async function onDOMContentLoaded({ frameId, tabId }) {
+    if (frameId || tabId !== authTabId) return;
+
     if (interactive) {
-      updateWindow(windowId, tabId, {focused: true, state: "normal"}).catch(err => console.error(err));
+      await updateWindow(authWindowId, authTabId, {
```
Comment (eight04) [inline index.js:85]: This is wrong. `updateWindow` shouldn't block the main process, Especially the cleanup at L92.

Also, it is better not to mix `.then` or `.catch` with `await`. Most of the time you can write
```js
await fn().catch(handleError);
```
as
```js
try {
  await fn();
} catch (err) {
  await handleError(err);
}
```

**Review by eight04 — CHANGES_REQUESTED** [review]
If the PR only includes "coding style" changes, I won't merge it. I only merge zero-feature refactoring if it can improve performance or maintainability.

**Comment by lucasrmendonca** [comment]
> If the PR only includes "coding style" changes, I won't merge it. I only merge zero-feature refactoring if it can improve performance or maintainability.

Ok I will try to open an issue instead of asking straight for pull requests in the future I think it'll be better

**Pull request closed without merging**